modules/Wordlist/README.md

```diff
@@ -10,17 +10,21 @@ thus in CLDF only a [`FormTable`](../../components/forms) is required.
+- Meaning specifications again should use reference catalogs like Concepticon. This may not be possible, though, in very specific dialect datasets where many cultural terms are of interest when studying language variation.
 
-Handling of the linguistic form is more complex and requires different layers of detail, starting from rough orthographic representations, up to sound sequence specifications which are cross-linguistically comparable. For the latter, we are developing standards within the [Cross-Linguistic Phonetic Notations (CLPN)](http://github.com/clpn) initative which aims to offer a framework that translates between closed transcriptions systems that have been in wide use, ranging from the coarse phonetic notation system of the [ASJP](http://asjp.clld.org) project, up to a [Cross-Linguistic Phonetic Alphabet (CLPA)](http://github.com/clpn/clpa) which we currently try to develop. In contrast to pure phonetic transcriptions, the CLPN initiative tries to standardize additional information, including the morphological segmentation of sound sequences, and a proper handling of suprasegmental features.
+It should be noted that both, Glottolog and Concepticon, offer ways to add new language varieties or concepts via contributions to the respective database. 
 
-In addition to the trias of language, form, and meaning, the word list specification of `cldf` also handles **relations** between the entities. Relations come in two general flavors: relations across languages, and relations within one language. The former are reflected in detailed specifications for the handling of different kinds of **cognacy** in a broader sense (akin to the concept of **homology** in biology, thus including borrowings, [List 2016](:bib:List2016i)). Language-internal relations point to language-internal cognacy, that is, form-associations within one language across different meanings, ranging from simple cases of **colexifications** up to fine-grained accounts on **internal reconstruction**. Language-internal relations require additional markup that indicates relations of forms within one language using different kinds of specification.  
+Handling of the linguistic form is more complex and requires different layers of detail, starting from rough orthographic representations, up to sound sequence specifications which are cross-linguistically comparable. For the latter, we are developing standards within the Cross-Linguistic Transcription Systems (CLTS) initative which aims to offer a framework that translates between closed transcriptions systems that have been in wide use, ranging from the coarse phonetic notation system of the [ASJP](http://asjp.clld.org) project, up to a broad IPA alphabet which we currently try to develop. In contrast to pure phonetic transcriptions, the CLTS initiative tries to standardize additional information, including the morphological segmentation of sound sequences, and a proper handling of suprasegmental features.
```

Comment:
If we add clts here, should we announce this as a first release, and maybe just make a release of the clts software already? Only code, not clld?

Reply:
So far, I mainly wanted to remove any mention of CLPA/CLPN. Whether we want to mention CLTS at this stage at all is an open question. I wouldn't want to rush a CLTS release just to be able to talk about it here.

Reply:
In this case, we should either completely discard it, or add a link, as the code is already in the open anyway, right?